{"uid":"54e71a772590651","name":"test_list_contacts_maxsize_negative","fullName":"test.Contactos.test_list_contacts#test_list_contacts_maxsize_negative","historyId":"175ebb84fb586afd8acd46fdc5e491e1","time":{"start":1726764909707,"stop":1726764909879,"duration":172},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError","statusTrace":"setup_teardown_list_contacts = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, {'accountIsInactive'...ccountsIds': [], ...}, {'accountIsInactive': None, 'accountName': None, 'accountsColumns': {}, 'accountsIds': [], ...})\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_list_contacts_maxsize_negative(setup_teardown_list_contacts):\n        headers, contact1, contact2 = setup_teardown_list_contacts\n        response = EspocrmRequest().get(EndpointContacts.list(maxSize=-1), headers)\n>       AssertionStatusCode().assert_status_code_400(response)\n\ntest/Contactos/test_list_contacts.py:107: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [500]>\n\n    @staticmethod\n    def assert_status_code_400(response):\n>       assert response.status_code == 400\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:19: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726764900558,"stop":1726764900558,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_teardown_list_contacts","time":{"start":1726764907405,"stop":1726764907847,"duration":442},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_teardown_list_contacts::0","time":{"start":1726764910972,"stop":1726764911337,"duration":365},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Contactos"},{"name":"suite","value":"test_list_contacts"},{"name":"host","value":"fv-az1020-822"},{"name":"thread","value":"2107-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Contactos.test_list_contacts"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["functional","regression","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"54e71a772590651.json","parameterValues":[]}